index.js

```diff
@@ -4,7 +4,7 @@ const _ = require('lodash')
 const cc = require('@conventional-commits/parser')
 const semver = require('semver')
 
-async function main () {
+async function main() {
```

Comment:
Don't change the code existing formatting, which follows Standard JS. Same for line 26.

Reply:
Sorry about that -- I didn't mean to change that. My editor did it automatically :). Should be back to the way it was.

Reply:
@NGPixel Ah, oops. Forgot the build step ... hold on a minute.

Reply:
@NGPixel Fixed now.